src/helpers.js

```diff
@@ -86,19 +92,35 @@ export function applySugar(environment: Object) {
   return environment
 }
 
+const pythonCode = 'import os;print("\0".join(map("=".join, dict(os.environ).items())))'
 export function getCommand(): { command: string, options: Object, parameters: Array<string> } {
+  const shellScript = coolTrim`
+    if [ "$(which python)" != "" ]; then
+      python -c '${pythonCode}'
+    elif [ "$(which python3)" != "" ]; then
+      python3 -c '${pythonCode}'
+    elif [ "$(which python2.7)" != "" ]; then
+      python2.7 -c '${pythonCode}'
+    else
+      env
```

Comment:
I would remove this. If you really need a fallback, use something like the script from #36.